docs/security.md

```diff
@@ -169,6 +175,12 @@ The following table describes the different options available for configuring th
   </td>
   <td>2.2.0</td>
 </tr>
+<tr>
+  <td><code>spark.network.crypto.authEngineVersion</code></td>
+  <td>1</td>
+  <td>Version of AES-based RPC encryption to use. Valid versions are 1 or 2. Version 2 is recommended.</td>
+  <td>4.0.0</td>
```

Comment:
@dongjoon-hyun , how do we handle the version here ?
We usually dont update new features - so calling it out as `4.0.0` for 3.5 (and 3.4) doc feels incorrect, thoughts ?

Reply:
Since we're backporting, we could put 3.4 there in all three versions, right?